couchdb/client.py

```diff
@@ -775,10 +775,161 @@ def put_attachment(self, doc, content, filename=None, content_type=None):
         }, rev=doc['_rev'])
         doc['_rev'] = data['rev']
 
+    def find(self, mango_query, wrapper=None):
+        """Execute a mango find-query against the database.
+
+        Note: Only available for CouchDB version >= 2.0.0 .
```

Comment:
Language nits: please start with lower-case characters after `:`, and remove the space before the period at the end (in this docstring as well as the others).